src/UI/Components/AvatarCircle/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Image, ImageSourcePropType, View } from 'react-native';
+import Svg, { Circle as SvgCircle } from 'react-native-svg';
```

Comment:
'react-native-svg': This package has been used but not present in package.json. Just cross check once.